### ISSUE TITLE
MoveState "ReadyToTrigger" not shown in this document

### ISSUE THREAD

**Issue by cimares** [MicrosoftDocs/office-docs-powershell#7744]
When executing the Get-SPOUserAndContentMoveState in our environment, we get a number of responses with a MoveState of ReadyToTrigger.

I'm assuming this means that the system has detected that the users PDL does not match their current GEO and that the system expects the user to have a migration performed at some point, but this is not made clear in this document.

![MoveState](https://user-images.githubusercontent.com/7425644/117675148-3b258680-b1a4-11eb-8db1-4eaaf2682223.jpg)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ea34b55f-8b6e-ad2e-b82c-574db46614a2
* Version Independent ID: 8fa169c0-74f3-92b0-4bcf-cb31bf616c9c
* Content: [Get-SPOUserAndContentMoveState (SharePointOnlinePowerShell)](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/get-spouserandcontentmovestate?view=sharepoint-ps&source=docs)
* Content Source: [sharepoint/sharepoint-ps/sharepoint-online/Get-SPOUserAndContentMoveState.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/sharepoint/sharepoint-ps/sharepoint-online/Get-SPOUserAndContentMoveState.md)
* Service: **sharepoint-powershell**
* GitHub Login: @trent-green
* Microsoft Alias: **trgreen**

**Comment by get-itips** [comment]
Hello @cimares Paul, how are you? Hope you are doing good.
Did you have chance to see [this other article?](https://docs.microsoft.com/en-us/microsoft-365/enterprise/move-sharepoint-between-geo-locations?view=o365-worldwide#determining-the-status-of-a-sharepoint-site-geo-move) with a description of the different values including Ready to trigger.
To my understanding the next move status should be Scheduled, did that happen for you?
Hope this information helps you

**Comment by cimares** [comment]
Just to be clear, we haven't issued the command to move any OneDrive sites yet. These are showing up as "Ready to Trigger" without us telling the Tenant to move the user.

If the system pre-preps OneDrives that have an incorrect PDL, this needs to be documented in the OneDrive move document (not the SharePoint site one) and the status should also be shown in this document.

**Comment by get-itips** [comment]
I understand better now @cimares Paul, thanks for the extra information.

Maybe these users already had an ODB site before setting their PDL?

> If the user already has a OneDrive site created in the tenant, setting their PDL will not automatically move their existing OneDrive.

https://docs.microsoft.com/en-us/microsoft-365/enterprise/multi-geo-tenant-configuration?view=o365-worldwide#onedrive-provisioning-and-the-effect-of-pdl

**Comment by cimares** [comment]
Sorry I think you're misunderstanding why I raised this issue. I know why the users have their OneDrives in the wrong region as we've just added a new Geo to the environment, We're getting ready to migrate these users into their final Geo.

This issue is specifically about the Documentation for this command not showing the status of "Ready to Trigger" as an available option (it doesn't exist in the description section which only shows "NotStarted", "InProgress", "Success", "Failed") nor explaining that you might find a bunch of your users already showing this status.

It would also be good to understand WHY they are showing as Ready To Trigger when we haven't used the Start-SPOUserAndContentMove cmdlet yet, especially given that this cmdlet has a limit of 1000 items returned, so if we try and return all active migrations, we don't see everything.
